inttest/common/footloosesuite.go

```diff
@@ -67,23 +73,27 @@ func (s *FootlooseSuite) SetupSuite() {
 	}
 	s.keyDir = dir
 	s.footlooseConfig = s.createConfig()
+
 	cluster, err := cluster.New(s.footlooseConfig)
+
 	if err != nil {
 		s.T().Logf("ERROR: failed to load footloose config: %s", err.Error())
 		s.T().FailNow()
 		return
 	}
 
+	// XXX: why do we create, delete and create again?
```

Comment:
If there happens to be some leftover container from previous runs of the same suite lingering around

Reply:
I changed it to be a explanation why do we do it